### PR TITLE
Task/default test fix

### PR DESCRIPTION
**Pull request by BenAAndrew**
## Description
Changes made for this branch

## Checklist
- [x] All manual tests passing
- [x] All unit tests passing 
- [x] All UI tests passing
- [ ] Code reviewed by peer:\
&nbsp; - *No redundant whitespace*\
&nbsp; - *Indentation correct*\
&nbsp; - *Naming conventions followed*\
&nbsp; - *Comments added (where needed)*\
&nbsp; - *No commented out redundant code*\
&nbsp; - *Branch name correct*
- [ ] Code reviewed by TA

### CODE REVIEW

File: Angular/src/app/add-capability/add-capability.component.spec.ts [hunk 1]
```diff
@@ -1,14 +1,21 @@
...
   let fixture: ComponentFixture<AddCapabilityComponent>;
 
   beforeEach(async(() => {
     TestBed.configureTestingModule({
-      declarations: [ AddCapabilityComponent ]
+      declarations: [ AddCapabilityComponent ],
+      imports: [ ReactiveFormsModule ],
+      providers: [ NgbActiveModal,
+                   DataService 
+                 ]
```
Comment (jamescrone) [inline Angular/src/app/add-capability/add-capability.component.spec.ts]: can this be brought onto the same line for readability. Please fix elsewhere in code

File: Angular/src/app/add-role-modal/add-role-modal.component.spec.ts [hunk 2]
```diff
@@ -1,14 +1,21 @@
...
   let fixture: ComponentFixture<AddRoleModalComponent>;
 
   beforeEach(async(() => {
     TestBed.configureTestingModule({
-      declarations: [ AddRoleModalComponent ]
+      declarations: [ AddRoleModalComponent ],
+      imports: [ ReactiveFormsModule ],
+      providers: [ NgbActiveModal,
+                   DataService
+                 ]
```
Comment (jamescrone) [inline Angular/src/app/add-role-modal/add-role-modal.component.spec.ts]: same here

File: Angular/src/app/admin-table/admin-table.component.spec.ts [hunk 3]
```diff
@@ -1,14 +1,31 @@
...
+                      NavbarComponent
+                    ],
+     imports: [ NgbAlertModule,
+                RouterTestingModule
+              ],
+     providers: [ AuthGuardComponent,
+                  AdminAuthGuardComponent,
+                  DataService,
+                  CookieService
+                ]
```
Comment (jamescrone) [inline Angular/src/app/admin-table/admin-table.component.spec.ts]: same here, " providers: [ AuthGuardComponent, AdminAuthGuardComponent, DataService, CookieService] " - example of standard format.

File: Angular/src/app/app.component.spec.ts [hunk 4]
```diff
@@ -1,10 +1,17 @@
...
 
 describe('AppComponent', () => {
   beforeEach(async(() => {
     TestBed.configureTestingModule({
+      providers:[
+        ChildrenOutletContexts
+      ],
       declarations: [
+        RouterOutlet,
+        NavbarComponent,
```
Comment (jamescrone) [inline Angular/src/app/app.component.spec.ts]: same here

**Comment by jamescrone** [comment]
*Can ignore these comments if this is how it's laid out in your notes.

File: Angular/src/app/add-capability/add-capability.component.spec.ts (hunk 1 above)
Comment (matthewdoreilly) [reply Angular/src/app/add-capability/add-capability.component.spec.ts]: I prefer the formatting like this: https://github.com/sapphireduffy/KainosCareerLattice/pull/39/files#diff-5ecc1ba348aae4edd8273433f83d029aR17

File: Angular/src/app/add-role-modal/add-role-modal.component.spec.ts (hunk 2 above)
Comment (matthewdoreilly) [reply Angular/src/app/add-role-modal/add-role-modal.component.spec.ts]: I prefer the formatting like this: https://github.com/sapphireduffy/KainosCareerLattice/pull/39/files#diff-5ecc1ba348aae4edd8273433f83d029aR17

File: Angular/src/app/admin-table/admin-table.component.spec.ts (hunk 3 above)
Comment (matthewdoreilly) [reply Angular/src/app/admin-table/admin-table.component.spec.ts]: I prefer the formatting like this: https://github.com/sapphireduffy/KainosCareerLattice/pull/39/files#diff-5ecc1ba348aae4edd8273433f83d029aR17

File: Angular/src/app/landing-page/landing-page.component.spec.ts [hunk 5]
```diff
@@ -1,14 +1,26 @@
...
     TestBed.configureTestingModule({
-      declarations: [ LandingPageComponent ]
+      declarations: [ LandingPageComponent,
+                      NavbarComponent
+                    ],
+      imports: [ RouterTestingModule ],
+      providers: [ AdminAuthGuardComponent,
+                   AuthGuardComponent,
+                   CookieService
+                 ]
```
Comment (matthewdoreilly) [inline Angular/src/app/landing-page/landing-page.component.spec.ts]: I prefer the formatting like this: https://github.com/sapphireduffy/KainosCareerLattice/pull/39/files#diff-5ecc1ba348aae4edd8273433f83d029aR17

File: Express-API/band.js [hunk 6]
```diff
@@ -1,9 +1,32 @@
...
+            try {
+                db.query(addBandDescriptionQuery,[params.commercialAwareness,params.communicatingAndTeamwork,
+                params.innovatioAndContinuousImprovement,params.customerFocus,
+                params.developingYourselfAndOthers,params.planningAndOrganising,
+                params.jobSpecificKnowledge]).then(rows => {
+                    db.query(addBandQuery,[params.name, params.schoolId, rows.insertId,
+                    params.training, params.responsibilities]).then(result => {
+                        resolve({"success":"Successfully added band"})
+                    })
+                })
```
Comment (matthewdoreilly) [inline Express-API/band.js]: I'd reformat this like:

```
try {
     db.query(addBandDescriptionQuery,
           [
                params.commercialAwareness, params.communicatingAndTeamwork,
                params.innovatioAndContinuousImprovement, params.customerFocus,
                params.developingYourselfAndOthers, params.planningAndOrganising,
                params.jobSpecificKnowledge
          ]
     ).then(rows => {
            ...
```

Just makes it more readable

**Pull request merged**